tutorials/timefreq/wavelets_tutorial.py

```diff
@@ -0,0 +1,166 @@
+# It simulates an oscillation, and takes input parameters n_seconds, fs, freq,
+# cycle, phase, and cycle_params.
+# The cycle is specified as sine, and the phase default is 0.
+# The cycle_params are the parameters for the simulated oscillation cycle,
+# which are already stored.
+#
+# We'll also need the function 'create_times' to create an array of time indices.
+# It takes input parameters n_seconds, fs, and start_val.
+# The n_seconds parameter is the length of the signal.
+# The fs parameter is the sampling frequency.
+# The start_val parameter is the starting value for the time definition, and it
+# has a default value of 0.
+
+###################################################################################################
+
+# Simulation settings
+fs = 500
+n_seconds = 10
+
+# Define simulation components
+components = {'sim_powerlaw': {exponent=-2.0, f_range=None, **filter_kwargs}, 'sim_oscillation' : {'freq': 10}})
+
+# Simulate a signal with a power-law time series with oscillations at 10 Hz.
+sig = sim_combined(n_seconds, fs, components)
+times = create_times(n_seconds, fs)
+
+###################################################################################################
+
+# Plot the simulated data
+plot_time_series(times, sig, 'Simulated EEG')
+
+###################################################################################################
+#
+# In the simulated signal above, we can see the time series data.
+#
+
+###################################################################################################
+# Compute Wavelet Transform Algorithm
+# ----------------------------------
+#
+# Now, lets use the Compute Morlet Wavlet Transform algorithm to transform our simulated time-series Data
+# to a time-frequency representation using morlet wavelets.
+# The algorithm computes the continuous morlet wavelet transform
+# at the specified frequencies and across all shifts.
+#
+# It takes parameters sig, fs, freqs, n_cycles, scaling, and norm.
+# The sig parameters is the time series signal.
+# The fs parameter is the sampling rate.
+# The freqs parameter defines the frequencies values or frequency range to estimate
+# with morlet wavelets.
+# The n_cycles parameter defines the number of cycles for each frequency.
+# The scaling parameter is the scaling factor for the morlet transform algorithm.
+# The norm parameter is optional, and defines the normalization method for the algorithm.
+# It can be specified as 'sss', or as 'amp'.
+# Specifying 'sss' divides by the square root of the sum of the squares.
+# Specifying 'amp' divides by the sum of the amplitudes.
+#
+###################################################################################################
+
+# Settings for the wavelet transform Algorithm
+freqs=[1, 30]
+
+# Compute wavelet transform using compute morlet wavelet transform algorithm
+mwt = compute_wavelet_transform(sig, fs=500, freqs)
```

Comment:
You can't have a positional argument (without an equals sign) after a keyword argument (`fs=500`) it's just not allowed in python. Best practice, again for user-facing code is to use keywords for everything unless it's totally obvious like `sig` (many times the first argument is totally obvious)

Reply:
`mwt = compute_wavelet_transform(sig, fs=500, freqs=freqs)` computes for me